packages/mermaid/src/mermaid.ts

```diff
@@ -337,6 +337,32 @@ const parse = async (text: string, parseOptions?: ParseOptions): Promise<boolean
   });
 };
 
+/**
+ * Function that renders an svg with a graph from a chart definition. Usage example below.
+ *
+ * ```javascript
+ * mermaid.initialize({
+ *   startOnLoad: true,
+ * });
+ * $(function () {
+ *   const graphDefinition = 'graph TB\na-->b';
+ *   const cb = function (svgGraph) {
+ *     console.log(svgGraph);
+ *   };
+ *   mermaid.render('id1', graphDefinition, cb);
```

Comment:
is this example correct?
`mermaid.render` no longer accepts a callback.

Reply:
No it's not. Thanks for spotting it.